doc/doxygen/doxyfile.in

```diff
@@ -303,7 +315,6 @@ EXPAND_ONLY_PREDEF     = YES
 SEARCH_INCLUDES        = YES
 INCLUDE_PATH           =
 INCLUDE_FILE_PATTERNS  = *.h
-#PREDEFINED            = protected=private \
 PREDEFINED =           = "HAVE_QHULL=1" \
```

Comment:
Could you also remove the first `=`, which seems to cause `error: Illegal PREDEFINED format '=', no define name specified`

Reply:
Used this required change to rebased the MR now.